jobs/active_circle_repos.rb

```diff
@@ -0,0 +1,21 @@
+require 'dashing'
+require File.expand_path('../../lib/circle_backend', __FILE__)
+
+# This job runs once a day to store the list of active Circle CI repos in an
+# env var. That way, it can be reused by the Circle CI widget without having
+# to make hundreds of API calls.
+#
+# The Circle CI API does not currently provide a way to get a list of just
+# the active repos, so we have to use the GitHub API to get a list of all
+# repos that belong to an organization, then make a separate Circle API call
+# for each one (over 300 for 18F), just to determine whether Circle CI has
+# builds for that repo.
```

Comment:
That could be another option, but I think what we have now is good enough. Eventually, once all repos have an `.about.yml`, we'll use the Team API to store a list of testable repos and only query those.